app/views/pages/_topics.html.erb

```diff
@@ -1,69 +1,64 @@
-<div class="grid-row dgu-topics">
-  <div class="column-one-third">
-    <div>
-      <ol>
-        <li>
-          <h2><%= link_to t('.business_and_economy_label'), search_path(filters: { topic: 'Business and economy' }), class: 'govuk-link' %></h2>
-          <p><%= t('.business_and_economy_description') %></p>
-        </li>
-        <li>
-          <h2><%= link_to t('.crime_and_justice_label'), search_path(filters: { topic: 'Crime and justice' }), class: 'govuk-link' %></h2>
-          <p><%= t('.crime_and_justice_description') %></p>
-        </li>
-        <li>
-          <h2><%= link_to t('.defence_label'), search_path(filters: { topic: 'Defence' }), class: 'govuk-link' %></h2>
-          <p><%= t('.defence_description') %></p>
-        </li>
-        <li>
-          <h2><%= link_to t('.education_label'), search_path(filters: { topic: 'Education' }), class: 'govuk-link' %></h2>
-          <p><%= t('.education_description') %></p>
-        </li>
-      </ol>
-    </div>
+<div class="govuk-grid-row">
+  <h2 class="govuk-visually-hidden"><%= t('.topics_heading') %></h2>
+  <div class="govuk-grid-column-one-third">
+    <ul class="govuk-list">
```

Comment:
This isn't a blocker - the (recently added) [list component](https://components.publishing.service.gov.uk/component-guide/list) could be used here.

Reply:
Discussed on slack. Because there's a need to add column styling for the `ul` I'm going to build this using markup and classes.